CONTRIBUTING.md

```diff
@@ -1,7 +1,14 @@
+# Contributing
+
 ## Process Guidelines
 
 While we're working quickly, please abide by a few rules to keep us safe and sane:
 * Enable 2FA for your account, and use hardware keys if possible (mandatory if you are a committer)
```

Comment:
```suggestion
* Enable 2FA for your account, and use hardware keys if possible (mandatory if you are a committer)
* Assign issues to yourself before you begin work to avoid duplicate effort.  If someone else already has the issue, please sync up before-hand.
* Assign PRs to the one or more reviewers whose approval you require.  All other reviewers are nice-to-have.
* If you are blocked on someone else, be bold and assign the issue directly to them so they are aware.
* If you have a PR assigned to you, consider it a high priority as you are likely blocking your colleague.
```